### PR TITLE
MFA Documentation

### PR DESCRIPTION
**Pull request by jault3**

### CODE REVIEW

File: guides/enable-multi-factor-auth.md [hunk 1]
```diff
@@ -0,0 +1,93 @@
+---
+title: Enable Multi-Factor Authentication
+category: guide
+---
+
+Security has always been a core principle at Catalyze. We've kept our user's best interests in mind when designing our products and how we can keep our users, and their data, safe. With the version 2.1 release of Stratum, multi-factor authentication (MFA) is readily available across all Catalyze products!
```
Comment (forana) [inline guides/enable-multi-factor-auth.md]: are version/release numbers public?

**Comment by rr1000** [comment]
I haven't opened this PR yet but I've added summary data to the frontmatter on all docs. It's just a simple one sentence explanation of the article. This is primarily used in the meta tags for OG, Twitter, and Swiftype.

**Comment by forana** [comment]
LGTM

File: guides/enable-multi-factor-auth.md [hunk 2]
```diff
@@ -1,3 +1,93 @@
...
+
+Here you'll be prompted for your username and password as well as which type of MFA you would like to enable. Catalyze currently supports two types of codes: those sent by email and those generated by an authenticator app such as Google Authenticator. Type in your credentials and choose `Email`.
+
+![email_mfa](../images/email_mfa.png)
+
+Once you click `Activate`, you will receive a one time use code at the email address associated with your account. This code may take a minute or two to arrive. Once you receive your code, enter it at the displayed prompt.
+
+![complete_email_mfa](../images/complete_email_mfa.png)
+
+For now, leave the `Set as preferred` checkbox checked. This is explained in the [Preferred Mode](#preferred-mode) section below. After entering the code and clicking `Complete MFA Activation` you will be shown a success notification.
```
Comment (rr1000) [inline guides/enable-multi-factor-auth.md]: I wouldn't refer to them as codes. Unless we want to change that in the UI. I would define One Time Password as a pretext and then use the OTP acronym throughout the guide.

File: guides/enable-multi-factor-auth.md (hunk 1 above)
Comment (andrewawagner) [inline guides/enable-multi-factor-auth.md]: _We've kept our user's best interests in mind when designing our products_
Change to present tense

_and how we can keep our users, and their data, safe_
Seems a bit clunky. Maybe change to something like "and do our utmost to keep user's and their data safe."

File: guides/enable-multi-factor-auth.md [hunk 3]
```diff
@@ -0,0 +1,93 @@
...
+
+![org_dropdown](../images/org_dropdown.png)
+
+Then you can click on `Require` to start enforcing MFA for all members of that organization.
+
+![org_mfa](../images/org_mfa.png)
+
+# Troubleshooting
+
+authenticator lockout
```
Comment (andrewawagner) [inline guides/enable-multi-factor-auth.md]: More coming here?

**Comment by rr1000** [comment]
LGTM

**Pull request merged**